### PR TITLE
Add more info to nats operator tls setup

### PR DESCRIPTION
**Pull request by wallyqs**

### CODE REVIEW

File: nats-kubernetes/nats-cluster-and-cert-manager.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+First need to install the cert-manager component from [jetstack](https://github.com/jetstack/cert-manager):
```
Comment (ColinSullivan1) [inline nats-kubernetes/nats-cluster-and-cert-manager.md]: `First you need`

File: nats-kubernetes/nats-cluster-and-cert-manager.md [hunk 2]
```diff
@@ -150,17 +143,20 @@ nats-2   1/1     Running   0          3s    172.17.0.9    minikube   <none>
...
-: [1] 2019/05/08 22:35:11.192952 [INF] Starting http monitor on 0.0.0.0:8222
-: [1] 2019/05/08 22:35:11.192981 [INF] Listening for client connections on 0.0.0.0:4222
-: [1] 2019/05/08 22:35:11.192987 [INF] TLS required for client connections
-: [1] 2019/05/08 22:35:11.192989 [INF] Server is ready
-: [1] 2019/05/08 22:35:11.193123 [INF] Listening for route connections on 0.0.0.0:6222
-: [1] 2019/05/08 22:35:12.487758 [INF] 172.17.0.9:49444 - rid:1 - Route connection created
-: [1] 2019/05/08 22:35:13.450067 [INF] 172.17.0.10:46286 - rid:2 - Route connection created
-```
\ No newline at end of file
+ [1] 2019/05/08 22:35:11.192781 [INF] Starting nats-server version 1.4.1
```
Comment (ColinSullivan1) [inline nats-kubernetes/nats-cluster-and-cert-manager.md]: Old version...

File: nats-kubernetes/nats-cluster-and-cert-manager.md [hunk 3]
```diff
@@ -49,11 +48,10 @@ spec:
     secretName: nats-ca
 ```
 
-```text
-issuer.certmanager.k8s.io/nats-ca created
-```
+Now create the certs that will match the DNS name used by the clients to connect, in this case traffic is within Kubernetes so using the name `nats` which is backed up by a headless service (here is an [example](https://github.com/nats-io/k8s/blob/master/nats-server/nats-server-plain.yml#L24-L47) of sample deployment)
```
Comment (ColinSullivan1) [inline nats-kubernetes/nats-cluster-and-cert-manager.md]: `so we are using the name`

File: nats-kubernetes/nats-cluster-and-cert-manager.md [hunk 4]
```diff
@@ -51,11 +50,10 @@ spec:
...
```
Comment (wallyqs) [inline nats-kubernetes/nats-cluster-and-cert-manager.md]:
```suggestion
Now create the certs that will match the DNS name used by the clients to connect, in this case traffic is within Kubernetes so we are using the name `nats` which is backed up by a headless service (here is an [example](https://github.com/nats-io/k8s/blob/master/nats-server/nats-server-plain.yml#L24-L47) of sample deployment)
```

File: nats-kubernetes/nats-cluster-and-cert-manager.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # NATS Cluster and Cert Manager
 
+First you need to install the cert-manager component from [jetstack](https://github.com/jetstack/cert-manager):
```
Comment (wallyqs) [inline nats-kubernetes/nats-cluster-and-cert-manager.md]:
```suggestion
First we need to install the cert-manager component from [jetstack](https://github.com/jetstack/cert-manager):
```

**Pull request merged**